src/VerticalDivider/verticaldivider.css

```diff
@@ -0,0 +1,11 @@
+
+@each $color $tone in $base-colors {
+    .u-vertical-divider--#{$tone} {
+      border-right: 1px solid $color;
+    }
+  }
+
+
+.u-vertical-divider--dark {
+  border-right: 1px solid darken(map-get($foregroundColors, subtle), 5);
+}
```

Comment:
@willwarren64 This is going to conflict with the `tone` prop, they're applying the same css property. Is it possible to add this to the iteration above and darken the tone so we combine those together instead? Not sure if that is necessary to have those extra dark classes. What do you think?

Reply:
I added this to try to bring it more in line with the horizontal Divider component. I can move it up as you suggested. As for whether it's necessary I'm not totally sure, but @subpopular might have an opinion